content/applications/inventory_and_mrp/inventory/routes/strategies/storage_category.rst

```diff
@@ -0,0 +1,171 @@
+real-time storage capacity checks based on package types: e.g., crates, bins, boxes.
+
+.. important::
+   Enable the :guilabel:`Packages` feature in :menuselection:`Inventory app --> Configuration -->
+   Settings` to show the :guilabel:`Capacity by Package` tab.
+
+.. example::
+   Create putaway rules for pallet-stored items by creating the `High Frequency pallets` storage
+   category.
+
+   In the :guilabel:`Capacity by Package` tab, specify the number of packages for the designated
+   :guilabel:`Package Type` and set a maximum of `2.00` `Pallets` for a specific location.
+
+   .. image:: storage_category/storage-category.png
+      :align: center
+      :alt: Create a storage category on the page.
+
+.. _inventory/routes/assign-location:
+
+Assign to location
+==================
+
+Once the storage category is created, assign it to a location. Navigate to the location by going to
+:menuselection:`Inventory app --> Configuration --> Locations`, and select the location. Click
+:guilabel:`Edit` and select the created category in the :guilabel:`Storage Category` field.
+
+.. example::
+   Assign the `High Frequency pallets` storage category (which limits pallets stored at any location
+   to two pallets) to the `WH/Stock/pallets/PAL 1` sub-location.
+
+   .. image:: storage_category/location-storage-category.png
+      :align: center
+      :alt: When a Storage Category is created, it can be linked to a warehouse location.
+
+.. _inventory/routes/set-putaway-attribute:
+
+Putaway rule
+============
+
+With the :ref:`storage category <inventory/routes/define-storage>` and :ref:`location
+<inventory/routes/assign-location>` set up, create the :doc:`putaway rule <putaway>` by navigating
+to :menuselection:`Inventory app --> Configuration --> Putaway Rules`.
+
+Click the :guilabel:`Create` button to create the putaway rule. In the :guilabel:`Having Category`
+field of the new putaway rule form, select the storage category.
+
+.. example::
+   Continuing the example from above, the `High Frequency Pallets` storage category is assigned to
+   the putaway rule directing pallets of lemonade to locations with the `High Frequency Pallets`
+   storage category :ref:`assigned to them <inventory/routes/assign-location>`.
+
+   .. image:: storage_category/smart-putaways.png
+      :align: center
+      :alt: Storage Categories used in a variety of putaway rules.
+
+Use case: limit capacity by package
+===================================
+
+To limit the capacity of a storage location by a specific number of packages, :ref:`create a storage
+category with a Capacity By Package <inventory/routes/set-capacity-package>`. Continuing the example
+from above, the `High Frequency Pallets` storage category is assigned to the `PAL1` and `PAL2`
+locations. Then, :ref:`putaway rules <inventory/routes/putaway-rule>` are set so that any pallets
+received in the warehouse are directed to be stored in `PAL1` and `PAL2` locations.
```

Comment:
```suggestion
category with a Capacity By Package <inventory/routes/set-capacity-package>`.

Continuing the example from above, the `High Frequency Pallets` storage category is assigned to the
`PAL1` and `PAL2` locations.

Then, :ref:`putaway rules <inventory/routes/putaway-rule>` are set, so that any pallets
received in the warehouse are directed to be stored in `PAL1` and `PAL2` locations.
```